scripts/ci-generate-publish-metadata.sh

```diff
@@ -13,19 +15,19 @@ set -euo pipefail
 
 # Generate the Vector version, and build description.
 VERSION="${VERSION:-"$(awk -F ' = ' '$1 ~ /^version/ { gsub(/["]/, "", $2); printf("%s",$2) }' Cargo.toml)"}"
-echo "name=vector_version::${VERSION}" >> "$GITHUB_OUTPUT"
+echo "vector_version::${VERSION}" >> "${GITHUB_OUTPUT}"
```

Comment:
```suggestion
echo "vector_version=${VERSION}" >> "${GITHUB_OUTPUT}"
```

Reply:
:laughing: attention to detail is key here